homeassistant/components/flo/config_flow.py

```diff
@@ -50,13 +49,11 @@ async def async_step_user(self, user_input=None):
         if user_input is not None:
             try:
                 info = await validate_input(self.hass, user_input)
-
+                await self.async_set_unique_id(user_input[CONF_USERNAME])
```

Comment:
We could move this up before validating the input and potentially save some time. Setting unique id and aborting should also be moved outside the try... except block.

Reply:
Done